src/tests/cypress/integration/service_transactions_statistics.spec.e2e.js

```diff
@@ -0,0 +1,53 @@
+/* global cy */
+describe('Gateway account statistics page', () => {
+  it('Renders today, week, month statistics successfully', () => {
+    cy.visit('/')
+
+    cy.get('a[href="/services"]').click()
+    
+    cy.contains('Mock').click()
+
+    cy.get('a[href="/gateway_accounts/1"]').click()
+    cy.get('.govuk-button').contains('View statistics').click()
+    cy.get('a[href="/transactions/statistics/?period=today&account=1"]')
```

Comment:
In general it might be a good idea to use either html `id`s or `data-*` attributes to reference these elements. 

As this isn't a test to verify that this is on the `transactions/statistics` route it would be useful to reduce the number of variables that could become flakey if different pieces change. 

For example here the links could have `data-test` attribute and then be referenced with 
```js
cy.get('[data-test=today]').click() 
```

The less dependant tests are on structure/ CSS/ JS the less likely they'll be to flake.